blind75/two_sum.py

```diff
@@ -59,6 +59,17 @@ def solution3(self, nums: List[int], target: int) -> List[int]:
                 if nums[i] + nums[j] == target:
                     return [i,j]
         return []
+    
+    # Time complexity: O(n2), Space complexity: O(1)
+    def solution3(self, nums: List[int], target: int) -> List[int]:
```

Comment:
Consider integrating the empty list check into the existing `solution3` method to avoid redundancy. This can be done by adding the conditional check at the beginning of the existing method, thus eliminating the need for a completely new method that largely duplicates the code. [important]